source/handy_httpd/components/config.d

```diff
@@ -83,19 +83,9 @@ struct ServerConfig {
      * is spawned to manage websocket connections, separate from the main
      * worker pool.
      */
-    bool enableWebSockets;
+    bool enableWebSockets = false;
 
     static ServerConfig defaultValues() {
```

Comment:
Can you also mark this as `deprecated` since all the default values are now defined by `ServerConfig.init`?

Reply:
will do

Reply:
all set